orocos_kdl/src/utilities/error_stack.cxx

```diff
@@ -57,7 +57,11 @@ void IOTracePopStr(char* buffer,int size) {
         *buffer = 0;
         return;
     }
+#if defined(_WIN32)
+    strncpy_s(buffer,size,errorstack.top().c_str(),size);
+#else
     strncpy(buffer,errorstack.top().c_str(),size);
+#endif
```

Comment:
Any reason why not to use `strncpy_s` on all platforms?

Reply:
No all libraries (like glibc) define `strncpy_s`. Microsoft's compiler, however, insists on using it :shrug: